config_coordinator/README.md

```diff
@@ -2,12 +2,12 @@ WLS Prometheus Exporter Coordinator
 =====
 
 Each instance in a WLS cluster maintains the configuration for its
-[Prometheus Exporter](http://github.com/oracle/wls_exporter) locally. The coordinator allows them to share
+Prometheus Exporter locally. The coordinator allows them to share
```

Comment:
shouldn't "locally" be close to "maintains"?  Like "Each instance in a WLS cluster maintains the configuration locally for its Prometheus Exporter."

Reply:
Yes, maintains locally. Maintains what locally, the configuration or the configuration for its Prometheus Exporter? I think either option is ok, so I took your suggestion.

Reply:
Please change "Prometheus Exporter" to "WebLogic Monitoring Exporter." When I wrote that originally, we hadn't picked an official name.

Reply:
Done.